ui/src/components/overview/MobileClientCardView.js

```diff
@@ -142,7 +143,7 @@ class MobileClientCardView extends Component {
             />
           </Filter>
           <div className="form-group">
-            <CreateClient platforms={SUPPORTED_PLATFORMS} />
+            <CreateClient platforms={SUPPORTED_PLATFORMS} item={new MobileApp().toJSON()} />
```

Comment:
maybe just have a default value in `CreateClient` component, rather than pass it in?

Reply:
That's correct, however the `CreateClient` is going to be fully rewritten (no `platform selection`, no `appIdentifier`, etc) so I don't think it worths fixing this.

Reply:
ok

Reply:
that's right. I already removed the supported platforms param in my PR, which is based on this PR